src/ngraph/ngraph_compiler.cc

```diff
@@ -111,6 +114,7 @@ void Compiler::Infer(const SimpleBindArg* simplebind) {
   const auto& idx = graph_.indexed_graph();
   shapes_.resize(idx.input_nodes().size(), nnvm::TShape());
   dtypes_.resize(idx.input_nodes().size(), -1);
+  stypes_.resize(idx.input_nodes().size(), -1);
```

Comment:
use enum (default or undefined) to init

Reply:
It's a data member, right? We don't know the size until this function is executed?

Reply:
thanks, will make this change